api/v1beta1/flowcollector_types.go

```diff
@@ -417,6 +417,11 @@ type FlowCollectorFLP struct {
 	// conversation end timeout is the duration of time to wait from the last flow log to end a conversation
 	ConversationEndTimeout *metav1.Duration `json:"conversationEndTimeout,omitempty"`
 
+	//+kubebuilder:default:="5s"
+	// +optional
+	// conversation terminating timeout is the duration of time to wait from detected FIN flag to end a connection
+	ConversationTerminatingTimeout *metav1.Duration `json:"conversationTerminatingTimeout,omitempty"`
```

Comment:
I have a mixed opinion on this setting, while I get that it might be useful to fix conn track issues that we can expect (e.g. if there's an overall high latency in flow collection), I also think it makes conn-tracking sound more complex from a lambda user PoV ... and also because the name of this setting is vey close from the other one `ConversationEndTimeout`.
Wouldn't it be a good candidate for moving into the Debug section?

Reply:
Just to clarify my comment: I think ideally this setting should be an implementation detail more than something to be configured, because in a "normal situation" we shouldn't expect to have to wait a long time before terminating the conversation, ie. it's only due to situations where there's a high latency for collection that we make it tweakable.. is that correct? So that's why it looks more to me like a debug kind of setting.

Reply:
Both `conversationEndTimeout` and `conversationTerminatingTimeout` are implementation details. 
If you don't want to see these here I'm fine moving both to debug but as said in PM we are not mature enough to define default values that work best in every scenario.

I would like to avoid asking customer to update `debug` section to use supported features.

Reply:
see latest comment from @ronensc:
https://issues.redhat.com/browse/NETOBSERV-915?focusedId=22417163&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22417163